src/editor/controller/editor_controller.cpp

```diff
@@ -1286,7 +1286,13 @@ void editor_controller::mouse_motion(int x, int y, const bool /*browse*/,
 
 void editor_controller::touch_motion(int x, int y, const bool browse, bool update, map_location new_loc)
 {
-
+	(void) x;
+	(void) y;
+	(void) browse;
+	(void) update;
+	(void) new_loc;
+	
+	// Not implemented at all. Sorry, it's a very low priority for iOS port.
```

Comment:
I disabled editor compilation once, but ended up enabling it and hiding the button in GUI.
It's not a big deal, given total app size and that it has no external dependencies.
IMHO, it's better to keep the mainline changes to a minimum.